spec/validator.js

```diff
@@ -50,4 +50,14 @@ describe('Validator constructor', function() {
   it('should have a check method', function() {
     expect(validator.check).to.be.a.function;
   });
+
+  it('should handle undefined data', function() {
+    var validator = new Validator(undefined, { name: 'required' })
+    validator.check();
```

Comment:
There is some weird thing with this:
`expect(fn).to.throw(Error)`
its calling asserted method with wrong scope which screws up `this` inside validator instance and gives me this error:
`AssertionError: expected [Function: fails] to not throw 'Error' but 'TypeError: this._checkAsync is not a function' was thrown`.
I switched to a `fails()` method though.